core/src/Kokkos_Graph.hpp

```diff
@@ -148,10 +148,19 @@ Graph<ExecutionSpace> create_graph(ExecutionSpace ex, Closure&& arg_closure) {
   return rv;
 }
 
+template <class ExecutionSpace = DefaultExecutionSpace>
+std::enable_if_t<Kokkos::is_execution_space_v<ExecutionSpace>,
+                 Graph<ExecutionSpace>>
+create_graph(const ExecutionSpace& exec = ExecutionSpace{}) {
+  return create_graph(exec, [](const auto& /* root */) {});
+}
+
 template <
     class ExecutionSpace = DefaultExecutionSpace,
     class Closure = Kokkos::Impl::DoNotExplicitlySpecifyThisTemplateParameter>
-Graph<ExecutionSpace> create_graph(Closure&& arg_closure) {
+std::enable_if_t<!Kokkos::is_execution_space_v<std::decay_t<Closure>>,
```

Comment:
I'd prefer if we used `Impl::remove_cvref_t` instead of `std::decay_t`

Reply:
I'll do.

Reply:
Done, closing.